### ISSUE TITLE
No upcoming talk

### ISSUE THREAD

**Issue by fharper** [LeaVerou/talks-list#9]
If there are no upcoming talks, the UI keeps the "Loading talk list..." message. Should be great to keep the usual UI with a message about the fact there is no upcoming talk planned yet.

Putting this here if someone have time before me... Should be simple to fix.

**Comment by LeaVerou** [comment]
Yup, that’s a known issue, just zero time to fix it. I know it’s easy but I barely have time to breathe these days! Any PRs would be very welcome :)

**Comment by fharper** [comment]
No worries, I created an issue as I'm in the same situation as you! I wanted to interest other developers who may have time before us :)
